src/worker.js

```diff
@@ -50,136 +50,124 @@ self.Module = {
     // Optimized builds contain a .js.mem file which is loaded asynchronously;
     // this waits until that has finished before performing further setup.
     onRuntimeInitialized: function () {
-        try {
```

Comment:
The change here is removing this `try` block and delegating error handling to the native `onerror` handler.